prowler/lib/check/models.py

```diff
@@ -646,7 +653,6 @@ def load_check_metadata(metadata_file: str) -> CheckMetadata:
         check_metadata = CheckMetadata.parse_file(metadata_file)
     except ValidationError as error:
         logger.critical(f"Metadata from {metadata_file} is not valid: {error}")
-        # TODO: remove this exit and raise an exception
-        sys.exit(1)
+        raise error
```

Comment:
Why this? Are we sure that this is not going to break anything?

Reply:
Raising the exception is actually safer, it allows errors to be handled or tested, instead of abruptly stopping the whole process with `sys.exit(1)`.